modules/cmsis/Kconfig

```
@@ -28,4 +28,10 @@ config CMSIS_M_CHECK_DEVICE_DEFINES
 	help
 	  This options enables the validation of CMSIS configuration flags.
 
+config CMSIS_CORE_HAS_SYSTEM_CORE_CLOCK
+	bool
+	default n
+	help
+	  Enable this option if CMSIS SystemCoreClock symbols is available.
```

`default n` is redundant.

```suggestion
config CMSIS_CORE_HAS_SYSTEM_CORE_CLOCK
	bool
	help
	  Enable this option if CMSIS SystemCoreClock symbols is available.
```